### PR TITLE
Use the new heatlh-starter module

### PR DESCRIPTION
**Pull request by KevinJCross**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RPE-908

### Change description ###
Added the use of the auto-configured health starter module

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/draftstore/controllers/LivenessHealthCheckTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource("/database.properties")
+public class LivenessHealthCheckTest {
+
+    @LocalServerPort
+    private int port;
+
+    @Test
+    public void should_return_UP_for_liveness_check() {
```
Comment (timja) [inline src/integrationTest/java/uk/gov/hmcts/reform/draftstore/controllers/LivenessHealthCheckTest.java]: no point adding an integration check for the starter imo, its tested there and doesn't need a separate integration test

File: src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java [hunk 2]
```diff
@@ -69,4 +72,26 @@ public void should_be_able_to_delete_all_users_drafts_in_service() throws Except
         draftStoreClient.deleteAllUsersDrafts();
         assertThat(draftStoreClient.readDraftPage().data).isEmpty();
     }
+
+    @Test
+    public void should_return_UP_for_liveness_check() {
+        given()
+            .accept(APPLICATION_JSON_VALUE)
+            .when()
+            .get("http" + draftStoreUrl.substring("https".length()) + "/health/liveness")
```
Comment (timja) [inline src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java]: is this how the rest of the calls are made? normally we set RestAssured.some insecure setting
to allow invalid certs

Comment (KevinJCross) [reply src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java]: it was more about the protocol in front due to the health endpoint not being under port 443.

File: src/integrationTest/java/uk/gov/hmcts/reform/draftstore/controllers/LivenessHealthCheckTest.java (hunk 1 above)
Comment (KevinJCross) [reply src/integrationTest/java/uk/gov/hmcts/reform/draftstore/controllers/LivenessHealthCheckTest.java]: sure ... will remove.

File: src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java (hunk 2 above)
Comment (timja) [reply src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java]: it is under port 443 as well

**Pull request merged**